components/button/README.md

```diff
@@ -86,3 +86,7 @@ Floating workflow buttons behavior can be added by using the `<d2l-floating-butt
 
 - `always-float` (Boolean): Indicates to display buttons as always floating
 - `min-height` (String, default: `'500px'`): The minimum height of the viewport to display floating buttons at (where applicable). If viewport is less than `min-height`, buttons will never appear floating (unless `always-float` is used). If viewport is greater than `min-height` then buttons will float when applicable.
+
+## Future Enhancements
+
```

Comment:
Do we want to list preset buttons?

Reply:
Hadn't thought about it, but we could... like `<d2l-button-preset type="save">` type thing?

Reply:
Yep that's what I was thinking... though I hadn't thought much about the API.